fesvr/htif.cc

```diff
@@ -82,8 +82,14 @@ htif_t::~htif_t()
 
 void htif_t::start()
 {
-  if (!targs.empty() && targs[0] != "none")
-      load_program();
+  if (!targs.empty() && targs[0] != "none"){
+      try{
+        load_program();
+      } catch( const std::runtime_error& err){
```

Comment:
Use of spaces is erratic and inconsistent

Reply:
I've always had trouble with this. I ran this through https://codebeautify.org/cpp-formatter-beautifier for the next commit, if it still looks erratic & there is a specific formatting tool you use, let me know.